cryptoki/tests/basic.rs

```diff
@@ -2303,12 +2327,18 @@ fn sign_verify_sha512_hmac() -> TestResult {
         Attribute::Sensitive(true),
         Attribute::Sign(true),
         Attribute::Verify(true),
-        Attribute::KeyType(KeyType::GENERIC_SECRET),
-        Attribute::Class(ObjectClass::SECRET_KEY),
         Attribute::ValueLen(256.into()),
     ];
 
-    let private = session.generate_key(&Mechanism::GenericSecretKeyGen, &priv_key_template)?;
+    let private = session.generate_key(
+        // SoftHSM does not yet support SHA-n key gen mechanisms
```

Comment:
Could you add a link here to a softhsm issue or source code showing no support? This may be useful once they *do* implement the mechanism.

Reply:
Done!

Reply:
Hmm... I had hoped there's a ticket directly related to SHA-n key gens but... well, okay :sweat_smile:

Reply:
Unfortunately not. They don't seem to have any issues open for individual 3.0 mechanisms to be added, so this was the best (existing) issue I could find 😕

Reply:
So the new thing works for Kryoptic? Huh... cool that we have that! :sweat_smile: